src/test/java/com/microsoft/azure/relay/HybridConnectionListenerTest.java

```diff
@@ -164,8 +161,6 @@ public void connectMultipleClientsTest() throws URISyntaxException {
 		CompletableFuture<?>[] listenerCloseFutures = new CompletableFuture<?>[MAX_CONNECTIONS_COUNT];
 		for (int i = 0; i < MAX_CONNECTIONS_COUNT; i++) {
 			HybridConnectionChannel listenerConnection = (HybridConnectionChannel) listenerConnectFutures[i].join();
-			int idx = i;
```

Comment:
>idx [](start = 7, length = 3)

These were used in conjunction with the commented out tracing. Maybe we should get rid of that too.